docker/lifemonitor.Dockerfile

```diff
@@ -1,34 +1,52 @@
-
 FROM python:3.7-buster
 
+# Declare the default 'lifemonitor' user
+ARG LM_USER=lm
+
+# Install base requirements
 RUN apt-get update -q \
  && apt-get install -y --no-install-recommends \
         bash \
         postgresql-client-11 \
-        python3-psycopg2 \
-        python3-sqlalchemy \
  && apt-get clean -y && rm -rf /var/lib/apt/lists
 
-ENV PYTHONPATH=/usr/local/lib/python3.7/dist-packages:/usr/lib/python3/dist-packages
-
 # Create a user 'lm' with HOME at /lm
-RUN useradd -d /lm -m lm
+RUN useradd -d /${LM_USER} -m ${LM_USER}
 
-COPY --chown=lm:lm requirements.txt /lm
-RUN pip3 install --no-cache-dir -r /lm/requirements.txt
-
-WORKDIR /lm
-ENV FLASK_DEBUG=1 \
+# Update Environment
+ENV PYTHONPATH=/${LM_USER}:/usr/local/lib/python3.7/dist-packages:/usr/lib/python3/dist-packages:${PYTHONPATH} \
+    FLASK_DEBUG=1 \
     FLASK_RUN_HOST=0.0.0.0 \
     GUNICORN_WORKERS=1 \
-    GUNICORN_THREADS=2
+    GUNICORN_THREADS=2 \
+    REQUESTS_CA_BUNDLE=/etc/ssl/certs/ca-certificates.crt
 
-ENTRYPOINT /usr/local/bin/lm_entrypoint.sh
+# Copy requirements and certificates
+COPY --chown=${LM_USER}:${LM_USER} requirements.txt certs/*.crt /${LM_USER}/
+
+# Install requirements and install certificates
+RUN pip3 install --no-cache-dir -r /${LM_USER}/requirements.txt
```

Comment:
Can we move this line and the previous one (`COPY...`) up to before the `ENV` definition to avoid having to rebuild to reset the environment variables?

Reply:
Sure! Fixed via https://github.com/crs4/life_monitor/pull/19/commits/56b6e5bb5d025645db220a74426816265be895e8